core/templates/dev/head/components/entity-creation-services/collection-creation-backend-api.service.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
There's no code owners for this file, please add one.

Reply:
This PR already contains the change in the CODEOWNERS file, PTAL.

Reply:
Year :)